application/models/Survey.php

```diff
@@ -1256,4 +1257,31 @@ public function getsSurveyUrl()
         return $this->sSurveyUrl;
     }
 
+    /**
+     * @return Question[]
+     */
+    public function getQuotableQuestions()
+    {
+        $criteria=new CDbCriteria;
+        $criteria->select = 't.*';
+        $criteria->with=array(
+            'survey.groups',
+        );
+
+        $criteria->order ='`groups`.`group_order`, `t`.`question_order`';
```

Comment:
Just a question : this was tested on pgsql and mssql ? Maybe must use quoteColumnName ?  
````
$criteria->order ='`groups`.`group_order`, `t`.`question_order`';
````
 seems strange and only related to mysql ?